pkg/plugin/aggregation/update.go

```diff
@@ -146,7 +146,9 @@ func (u *updater) ReceiveAll(results map[string]*plugin.Result) {
 	}
 }
 
-func getPatch(annotation string) map[string]interface{} {
+// GetPatch takes a json encoded string and creates a map which can be used as
+// a patch to indicate the Sonobuoy status.
+func GetPatch(annotation string) map[string]interface{} {
```

Comment:
I did decide to export this; otherwise I was 100% duplicating this method in the discovery package. It didn't seem like there was really anything to gain from that seeing as how I did genuinely have a use case for this method outside of this pkg.